<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TraceEditDialog</class>
 <widget class="QDialog" name="TraceEditDialog">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>931</width>
    <height>392</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Edit Trace</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_5">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_6" stretch="0,1">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <layout class="QFormLayout" name="formLayout">
         <item row="0" column="0">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Name:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="name"/>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Color:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="ColorPickerButton" name="color">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>Velocity Factor:</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="SIUnitEdit" name="vFactor"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QRadioButton" name="bLive">
           <property name="text">
            <string>Live Capture</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">GSource</string>
           </attribute>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="bFile">
           <property name="text">
            <string>From File</string>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">GSource</string>
           </attribute>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>Reference impedance:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="SIUnitEdit" name="impedance">
           <property name="enabled">
            <bool>false</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QStackedWidget" name="stack">
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="LivePage">
          <layout class="QFormLayout" name="formLayout_2">
           <item row="0" column="0">
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>Type:</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QComboBox" name="CLiveType">
             <item>
              <property name="text">
               <string>Overwrite</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Max hold</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Min hold</string>
              </property>
             </item>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>Parameter:</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QComboBox" name="CLiveParam"/>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="TouchstonePage">
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="TouchstoneImport" name="touchstoneImport" native="true"/>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <widget class="QLabel" name="label_5">
               <property name="text">
                <string>Parameter:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="CParameter"/>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="CSVPage">
          <layout class="QVBoxLayout" name="verticalLayout_6">
           <item>
            <widget class="CSVImport" name="csvImport" native="true"/>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Math operations</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QTableView" name="view">
            <property name="horizontalScrollBarPolicy">
             <enum>Qt::ScrollBarAsNeeded</enum>
            </property>
            <property name="dragEnabled">
             <bool>true</bool>
            </property>
            <property name="dragDropMode">
             <enum>QAbstractItemView::InternalMove</enum>
            </property>
            <property name="selectionMode">
             <enum>QAbstractItemView::SingleSelection</enum>
            </property>
            <property name="selectionBehavior">
             <enum>QAbstractItemView::SelectRows</enum>
            </property>
            <property name="showGrid">
             <bool>false</bool>
            </property>
            <attribute name="horizontalHeaderMinimumSectionSize">
             <number>20</number>
            </attribute>
            <attribute name="horizontalHeaderDefaultSectionSize">
             <number>70</number>
            </attribute>
            <attribute name="horizontalHeaderStretchLastSection">
             <bool>false</bool>
            </attribute>
            <attribute name="verticalHeaderVisible">
             <bool>false</bool>
            </attribute>
           </widget>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <widget class="QPushButton" name="bAdd">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="toolTip">
               <string>Add</string>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="icon">
               <iconset theme="list-add" resource="../icons.qrc">
                <normaloff>:/icons/add.png</normaloff>:/icons/add.png</iconset>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="bDelete">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="toolTip">
               <string>Delete</string>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="icon">
               <iconset theme="list-remove" resource="../icons.qrc">
                <normaloff>:/icons/remove.png</normaloff>:/icons/remove.png</iconset>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="bMoveUp">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="toolTip">
               <string>Move up</string>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="icon">
               <iconset theme="go-up" resource="../icons.qrc">
                <normaloff>:/icons/up.png</normaloff>:/icons/up.png</iconset>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="bMoveDown">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="toolTip">
               <string>Move down</string>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="icon">
               <iconset theme="go-down" resource="../icons.qrc">
                <normaloff>:/icons/down.png</normaloff>:/icons/down.png</iconset>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="verticalSpacer">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>18</width>
                <height>186</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SIUnitEdit</class>
   <extends>QLineEdit</extends>
   <header>CustomWidgets/siunitedit.h</header>
  </customwidget>
  <customwidget>
   <class>TouchstoneImport</class>
   <extends>QWidget</extends>
   <header>CustomWidgets/touchstoneimport.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ColorPickerButton</class>
   <extends>QPushButton</extends>
   <header>CustomWidgets/colorpickerbutton.h</header>
  </customwidget>
  <customwidget>
   <class>CSVImport</class>
   <extends>QWidget</extends>
   <header>CustomWidgets/csvimport.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../icons.qrc"/>
 </resources>
 <connections/>
 <buttongroups>
  <buttongroup name="GSource"/>
 </buttongroups>
</ui>
